tap_hubspot/streams.py

```diff
@@ -522,3 +522,86 @@ class OwnersStream(HubSpotStream):
             ),
         ),
     ).to_dict()
+
+
+### Associations streams
+class DealsAssociationsStream(HubSpotStream):
+  """Deal's associations"""
+
+  name = "deals_associations"
+  path = "/crm/v4/objects/deal/?associations=companies,contacts&limit=100"
+  properties_object_type = "deals"
+  primary_keys = ["id"]
+  replication_key = None
+  schema = th.PropertiesList(
+    th.Property(
+      "id",
+      th.StringType,
+    ),
+    th.Property(
+      "updatedAt",
+      th.DateTimeType,
+    ),
+    th.Property(
+      "archived",
+      th.BooleanType,
+    ),
+    th.Property(
+      "associations",
+      th.StringType,
+    )
+  ).to_dict()
+
+class ContactAssociationsStream(HubSpotStream):
+  """Contact's associations."""
+
+  name = "contact_associations"
+  path = "/crm/v4/objects/contact/?associations=companies,deals&limit=100"
+  properties_object_type = "contacts"
+  primary_keys = ["id"]
+  replication_key = None
+  schema = th.PropertiesList(
+    th.Property(
+      "id",
+      th.StringType,
+    ),
+    th.Property(
+      "updatedAt",
+      th.DateTimeType,
+    ),
+    th.Property(
+      "archived",
+      th.BooleanType,
+    ),
+    th.Property(
+      "associations",
+      th.StringType,
+    )
+  ).to_dict()
+
+class CompanyAssociationsStream(HubSpotStream):
+  """Company's associations."""
+
+  name = "company_associations"
+  path = "/crm/v4/objects/company/?associations=contacts,deals&limit=100"
```

Comment:
remove `limit`

This is already added here: https://github.com/spacecowboy/tap-hubspot/blob/2ad50cdac9563d293ce1fd997627697dbe53e276/tap_hubspot/client.py#L120